### PR TITLE
--add: combine demos and challenges for array methods

### PR DESCRIPTION
**Pull request by onemanwenttomow**

### CODE REVIEW

File: sessions/js-array-methods/animal-objects/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Array Methods 2: An Array with Objects
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-objects/README.md:1]:
```suggestion
# Array Methods: An Array with Objects
```

File: sessions/js-array-methods/animal-objects/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 2: Array of Animal Objects</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-objects/index.html:6]:
```suggestion
    <title>JS Array Methods: Array of Animal Objects</title>
```

File: sessions/js-array-methods/animal-objects_solution/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Array Methods 2: An Array with Objects
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-objects_solution/README.md:1]:
```suggestion
# Array Methods: An Array with Objects
```

File: sessions/js-array-methods/animal-objects_solution/index.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 2: Array of Animal Objects</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-objects_solution/index.html:6]:
```suggestion
    <title>JS Array Methods: Array of Animal Objects</title>
```

File: sessions/js-array-methods/animal-objects_solution/package.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_animal-objects_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-objects_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_animal-objects_solution",
```

File: sessions/js-array-methods/animal-objects/package.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_animal-objects",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-objects/package.json:2]:
```suggestion
  "name": "js-array-methods_animal-objects",
```

File: sessions/js-array-methods/animal-strings/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+# Array Methods 2: A simple String Array
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-strings/README.md:1]:
```suggestion
# Array Methods: A simple String Array
```

File: sessions/js-array-methods/animal-strings/index.html [hunk 8]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 2: Array of Animal Strings</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-strings/index.html:6]:
```suggestion
    <title>JS Array Methods: Array of Animal Strings</title>
```

File: sessions/js-array-methods/animal-strings/package.json [hunk 9]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_animal-strings",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-strings/package.json:2]:
```suggestion
  "name": "js-array-methods_animal-strings",
```

File: sessions/js-array-methods/animal-strings_solution/README.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
+# Array Methods 2: A simple String Array
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-strings_solution/README.md:1]:
```suggestion
# Array Methods: A simple String Array
```

File: sessions/js-array-methods/animal-strings_solution/index.html [hunk 11]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 2: Array of Animal Strings</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-strings_solution/index.html:6]:
```suggestion
    <title>JS Array Methods: Array of Animal Strings</title>
```

File: sessions/js-array-methods/animal-strings_solution/package.json [hunk 12]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_animal-strings_solution",
+  "version": "0.0.0-unreleased",
+  "description": "Challenge for JS Array Methods 2 with Array of animal strings",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-strings_solution/package.json:4]:
```suggestion
  "description": "Challenge for JS Array Methods with Array of animal strings",
```

File: sessions/js-array-methods/animal-strings_solution/package.json [hunk 13]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_animal-strings_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/animal-strings_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_animal-strings_solution",
```

File: sessions/js-array-methods/employees/README.md [hunk 14]
```diff
@@ -0,0 +1,22 @@
+# JS Array Methods 2: Employees
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees/README.md:1]:
```suggestion
# JS Array Methods: Employees
```

File: sessions/js-array-methods/employees/index.html [hunk 15]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Employees</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees/index.html:9]:
```suggestion
    <title>JS Array Methods: Employees</title>
```

File: sessions/js-array-methods/employees/package.json [hunk 16]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_employees",
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2: Employees",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees/package.json:4]:
```suggestion
  "description": "JS Array Methods: Employees",
```

File: sessions/js-array-methods/employees_solution/README.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
+# JS Array Methods 2: Employees
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees_solution/README.md:1]:
```suggestion
# JS Array Methods: Employees
```

File: sessions/js-array-methods/employees_solution/index.html [hunk 18]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Employees</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees_solution/index.html:9]:
```suggestion
    <title>JS Array Methods: Employees</title>
```

File: sessions/js-array-methods/employees_solution/package.json [hunk 19]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_employees_solution",
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2: Employees",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees_solution/package.json:4]:
```suggestion
  "description": "JS Array Methods: Employees",
```

File: sessions/js-array-methods/employees_solution/package.json [hunk 20]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_employees_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_employees_solution",
```

File: sessions/js-array-methods/employees/package.json [hunk 21]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_employees",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/employees/package.json:2]:
```suggestion
  "name": "js-array-methods_employees",
```

File: sessions/js-array-methods/filter/index.html [hunk 22]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 1: Filter</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/filter/index.html:6]:
```suggestion
    <title>JS Array Methods: Filter</title>
```

File: sessions/js-array-methods/filter_solution/index.html [hunk 23]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 1: Filter</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/filter_solution/index.html:6]:
```suggestion
    <title>JS Array Methods: Filter</title>
```

File: sessions/js-array-methods/filter_solution/package.json [hunk 24]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-1_filter_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/filter_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_filter_solution",
```

File: sessions/js-array-methods/for-each/index.html [hunk 25]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>JS Array Methods 1: Color Boxes (forEach)</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/for-each/index.html:7]:
```suggestion
    <title>JS Array Methods: Color Boxes (forEach)</title>
```

File: sessions/js-array-methods/for-each_solution/index.html [hunk 26]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>JS Array Methods 1: Color Boxes (forEach)</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/for-each_solution/index.html:7]:
```suggestion
    <title>JS Array Methods: Color Boxes (forEach)</title>
```

File: sessions/js-array-methods/for-each_solution/package.json [hunk 27]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "js-array-methods-1_for-each_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/for-each_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_for-each_solution",
```

File: sessions/js-array-methods/fun-with-flags/index.html [hunk 28]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Fun with Flags</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/fun-with-flags/index.html:9]:
```suggestion
    <title>JS Array Methods: Fun with Flags</title>
```

File: sessions/js-array-methods/fun-with-flags/package.json [hunk 29]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "js-array-methods-2_fun-with-flags",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/fun-with-flags/package.json:2]:
```suggestion
  "name": "js-array-methods_fun-with-flags",
```

File: sessions/js-array-methods/fun-with-flags/package.json [hunk 30]
```diff
@@ -0,0 +1,21 @@
...
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2 Challenge: Fun with Flags",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/fun-with-flags/package.json:4]:
```suggestion
  "description": "JS Array Methods Challenge: Fun with Flags",
```

File: sessions/js-array-methods/fun-with-flags_solution/index.html [hunk 31]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Fun with Flags</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/fun-with-flags_solution/index.html:9]:
```suggestion
    <title>JS Array Methods: Fun with Flags</title>
```

File: sessions/js-array-methods/fun-with-flags_solution/package.json [hunk 32]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "js-array-methods-2_fun-with-flags_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/fun-with-flags_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_fun-with-flags_solution",
```

File: sessions/js-array-methods/fun-with-flags_solution/package.json [hunk 33]
```diff
@@ -0,0 +1,13 @@
...
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2 Challenge: Fun with Flags",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/fun-with-flags_solution/package.json:4]:
```suggestion
  "description": "JS Array Methods Challenge: Fun with Flags",
```

File: sessions/js-array-methods/letter-puzzle/README.md [hunk 34]
```diff
@@ -0,0 +1,24 @@
+# JS Array Methods 2: Employee Data Puzzle
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle/README.md:1]:
```suggestion
# JS Array Methods: Employee Data Puzzle
```

File: sessions/js-array-methods/letter-puzzle/index.html [hunk 35]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Letter Puzzle</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle/index.html:9]:
```suggestion
    <title>JS Array Methods: Letter Puzzle</title>
```

File: sessions/js-array-methods/letter-puzzle/package.json [hunk 36]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_letter-puzzle",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle/package.json:2]:
```suggestion
  "name": "js-array-methods_letter-puzzle",
```

File: sessions/js-array-methods/letter-puzzle/package.json [hunk 37]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2: Letter Puzzle",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle/package.json:4]:
```suggestion
  "description": "JS Array Methods: Letter Puzzle",
```

File: sessions/js-array-methods/letter-puzzle_solution/README.md [hunk 38]
```diff
@@ -0,0 +1,24 @@
+# JS Array Methods 2: Employee Data Puzzle
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle_solution/README.md:1]:
```suggestion
# JS Array Methods: Employee Data Puzzle
```

File: sessions/js-array-methods/letter-puzzle_solution/index.html [hunk 39]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Letter Puzzle</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle_solution/index.html:9]:
```suggestion
    <title>JS Array Methods: Letter Puzzle</title>
```

File: sessions/js-array-methods/letter-puzzle_solution/package.json [hunk 40]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_letter-puzzle_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_letter-puzzle_solution",
```

File: sessions/js-array-methods/letter-puzzle_solution/package.json [hunk 41]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2: Letter Puzzle",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/letter-puzzle_solution/package.json:4]:
```suggestion
  "description": "JS Array Methods: Letter Puzzle",
```

File: sessions/js-array-methods/map/index.html [hunk 42]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 1: Map</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/map/index.html:6]:
```suggestion
    <title>JS Array Methods: Map</title>
```

File: sessions/js-array-methods/map_solution/index.html [hunk 43]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 1: Map</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/map_solution/index.html:6]:
```suggestion
    <title>JS Array Methods: Map</title>
```

File: sessions/js-array-methods/map_solution/package.json [hunk 44]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-1_map_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/map_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_map_solution",
```

File: sessions/js-array-methods/news-flash/index.html [hunk 45]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: News Flash</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/news-flash/index.html:9]:
```suggestion
    <title>JS Array Methods: News Flash</title>
```

File: sessions/js-array-methods/news-flash/package.json [hunk 46]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "js-array-methods-2_news-flash",
+  "version": "0.0.0-unreleased",
+  "description": "News Flash Challenge for JS Array Methods 2",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/news-flash/package.json:4]:
```suggestion
  "description": "News Flash Challenge for JS Array Methods",
```

File: sessions/js-array-methods/news-flash/package.json [hunk 47]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "js-array-methods-2_news-flash",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/news-flash/package.json:2]:
```suggestion
  "name": "js-array-methods_news-flash",
```

File: sessions/js-array-methods/news-flash_solution/index.html [hunk 48]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: News Flash</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/news-flash_solution/index.html:9]:
```suggestion
    <title>JS Array Methods: News Flash</title>
```

File: sessions/js-array-methods/news-flash_solution/package.json [hunk 49]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "js-array-methods-2_news-flash_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/news-flash_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_news-flash_solution",
```

File: sessions/js-array-methods/news-flash_solution/package.json [hunk 50]
```diff
@@ -0,0 +1,13 @@
...
+  "version": "0.0.0-unreleased",
+  "description": "News Flash Challenge for JS Array Methods 2",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/news-flash_solution/package.json:4]:
```suggestion
  "description": "News Flash Challenge for JS Array Methods",
```

File: sessions/js-array-methods/repetition_solution/index.html [hunk 51]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS Array Methods 1: Map and Filter</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/repetition_solution/index.html:6]:
```suggestion
    <title>JS Array Methods: Map and Filter</title>
```

File: sessions/js-array-methods/repetition_solution/package.json [hunk 52]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-1_repetition_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/repetition_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_repetition_solution",
```

File: sessions/js-array-methods/spa-1/index.html [hunk 53]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 1: Single Page Application 1</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/spa-1/index.html:9]:
```suggestion
    <title>JS Array Methods: Single Page Application 1</title>
```

File: sessions/js-array-methods/spa-2/index.html [hunk 54]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 1: Single Page Application 2</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/spa-2/index.html:9]:
```suggestion
    <title>JS Array Methods: Single Page Application 2</title>
```

File: sessions/js-array-methods/spa-3/index.html [hunk 55]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 1: Single Page Application 3</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/spa-3/index.html:9]:
```suggestion
    <title>JS Array Methods: Single Page Application 3</title>
```

File: sessions/js-array-methods/zoo-animals/README.md [hunk 56]
```diff
@@ -0,0 +1,20 @@
+# JS Array Methods 2: Zoo Animals
```
Comment (JessicaLoers) [inline sessions/js-array-methods/zoo-animals/README.md:1]:
```suggestion
# JS Array Methods: Zoo Animals
```

File: sessions/js-array-methods/zoo-animals/index.html [hunk 57]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Zoo Animals</title>
```
Comment (JessicaLoers) [inline sessions/js-array-methods/zoo-animals/index.html:9]:
```suggestion
    <title>JS Array Methods: Zoo Animals</title>
```

File: sessions/js-array-methods/zoo-animals/package.json [hunk 58]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_zoo-animals",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/zoo-animals/package.json:2]:
```suggestion
  "name": "js-array-methods_zoo-animals",
```

File: sessions/js-array-methods/zoo-animals/package.json [hunk 59]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2: Zoo Animals",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/zoo-animals/package.json:4]:
```suggestion
  "description": "JS Array Methods: Zoo Animals",
```

File: sessions/js-array-methods/zoo-animals_solution/README.md [hunk 60]
```diff
@@ -0,0 +1,20 @@
+# JS Array Methods 2: Zoo Animals
```
Comment (JessicaLoers) [inline sessions/js-array-methods/zoo-animals_solution/README.md:1]:
```suggestion
# JS Array Methods: Zoo Animals
```

File: sessions/js-array-methods/zoo-animals_solution/package.json [hunk 61]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-array-methods-2_zoo-animals_solution",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/zoo-animals_solution/package.json:2]:
```suggestion
  "name": "js-array-methods_zoo-animals_solution",
```

File: sessions/js-array-methods/zoo-animals_solution/package.json [hunk 62]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "0.0.0-unreleased",
+  "description": "JS Array Methods 2: Zoo Animals",
```
Comment (JessicaLoers) [inline sessions/js-array-methods/zoo-animals_solution/package.json:4]:
```suggestion
  "description": "JS Array Methods: Zoo Animals",
```

**Comment by onemanwenttomow** [comment]
> "Just" removed session numbers for consistency 🫨

 "Just" 🤣
 
 Wow. Nice find Jessica!

**Pull request merged**